### PR TITLE
Hanlde the image upload functionality

### PR DESCRIPTION
**Pull request by TrNgTien**

### CODE REVIEW

File: src/store/actions/posting-project-user/profile/CreatePageAction.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+        const imageUrl = await handleUpLoad(imageFile);
+
+        console.log('url in createProfile', imageUrl)
+
+        try {
+            await
+            firebase_db
+            .collection('organization')
+            .doc(`${testing_organization_id}`)
+            .set({
```
Comment (kenpham4real) [inline src/store/actions/posting-project-user/profile/CreatePageAction.js]: `set` method is used for changing the document. We'll accept the code in the situation where deadline is a priority.

--> Notice that, the task is to **add** (aka create a **new document**). Please take time research on this _(keyword: **add** method of firebase document)_

**Review by kenpham4real — COMMENTED** [review]
Great hustle.

File: src/store/actions/posting-project-user/profile/CreatePageAction.js (hunk 1 above)
Comment (TrNgTien) [reply src/store/actions/posting-project-user/profile/CreatePageAction.js]: From [Firebase doc](https://firebase.google.com/docs/firestore/manage-data/add-data) 

> Behind the scenes, .add(...) and .doc().set(...) are completely equivalent, so you can use whichever is more convenient.

Comment (TrNgTien) [reply src/store/actions/posting-project-user/profile/CreatePageAction.js]: From[StackOverFlow](https://stackoverflow.com/questions/47474522/firestore-difference-between-set-and-add#:~:text=When%20you%20use%20set(),identified%20by%20some%20unique%20id.&text=When%20you%20use%20add(),unique%20id%20assigned%20to%20it.)

**Pull request merged**